galaxy.yml

```
@@ -7,13 +7,15 @@ readme: README.md
 authors:
   - Jonathan Davila (!UNKNOWN) <jdavila(at)ansible.com>
```

```suggestion
  - Jonathan Davila
```
That specific format is needed for plugins/modules, but not here :)